meta-resin-common/recipes-core/healthdog/healthdog_0.1.0.bb

```diff
@@ -0,0 +1,52 @@
+# Auto-Generated by cargo-bitbake 0.3.7
+#
+inherit cargo
+
+# If this is git based prefer versioned ones if they exist
+# DEFAULT_PREFERENCE = "-1"
+
+# how to get healthdog could be as easy as but default to a git checkout:
+# SRC_URI += "crate://crates.io/healthdog/0.1.0"
+SRC_URI += "git://git@github.com/resin-os/healthdog-rs.git;protocol=ssh"
+SRCREV = "a10155844f251d00ec49348e58b2846803386085"
+S = "${WORKDIR}/git"
+CARGO_SRC_DIR=""
+PV_append = ".AUTOINC+a10155844f"
+
+# please note if you have entries that do not begin with crate://
+# you must change them to how that package can be fetched
+SRC_URI += " \
+	crate://crates.io/bitflags/0.7.0 \
+	crate://crates.io/cfg-if/0.1.1 \
+	crate://crates.io/errno/0.2.3 \
+	crate://crates.io/exec/0.3.1 \
+	crate://crates.io/getopts/0.2.14 \
+	crate://crates.io/kernel32-sys/0.2.2 \
+	crate://crates.io/libc/0.2.24 \
+	crate://crates.io/libsystemd-sys/0.1.0 \
+	crate://crates.io/log/0.3.8 \
+	crate://crates.io/mbox/0.3.1 \
+	crate://crates.io/nix/0.8.1 \
+	crate://crates.io/pkg-config/0.3.9 \
+	crate://crates.io/rustc_version/0.2.1 \
+	crate://crates.io/semver-parser/0.7.0 \
+	crate://crates.io/semver/0.6.0 \
+	crate://crates.io/systemd/0.1.0 \
+	crate://crates.io/utf8-cstr/0.1.6 \
+	crate://crates.io/void/1.0.2 \
+	crate://crates.io/winapi-build/0.1.1 \
+	crate://crates.io/winapi/0.2.8 \
```

Comment:
NIT all the dependencies are in the same order as in https://github.com/resin-os/healthdog-rs/blob/master/Cargo.lock, except the last two (`winapi` and `winapi-build`) which is in flipped order.

Reply:
according to the header this file was autogenerated. @agherzan can you make sure this file is the exact output of carg-bitbake?